cms_form/models/cms_form_mixin.py

```diff
@@ -324,15 +324,23 @@ def form_get_request_values(self):
             # and this will make the form machinery miss all the fields
             _values = self.request.args
         # normal fields
-        values = {
-            k: v for k, v in _values.items()
-            if k not in ('csrf_token', )
-        }
+        res = {}
+        for k, v in _values.items():
+            if k in ('csrf_token', ):
+                continue
+            if k.endswith(':list'):
+                # fields w/ multiple values
+                # TODO
+                # 1. add test and docs
+                # 2. support more "transformers" (`:int` for instance)
+                v = _values.getlist(k)
```

Comment:
i think we should skip this if we have already processed an item of this list.

Reply:
the funny thing is that "dictionary-wise" you get only one key, so if you iter or access the key you get only one. That's why you must use getlist. See http://werkzeug.pocoo.org/docs/0.14/datastructures/#werkzeug.datastructures.MultiDict ;)

Reply:
Interesting, didn't know that.